server/core/config/raw/metrics_test.go

```diff
@@ -122,15 +131,6 @@ func TestMetrics_Validate_Error(t *testing.T) {
 				},
 			},
 		},
-		{
-			description: "invalid host",
-			subject: raw.Metrics{
-				Statsd: &raw.Statsd{
-					Host: "127.0.1",
-					Port: "8125",
-				},
-			},
-		},
```

Comment:
I deleted this test case, because:

"127.0.1" is an invalid IP address, it is also invalid hostname.

In RFC 1034 3.5 Preferred name syntax, a host name is defined as follows:

> The labels must follow the rules for ARPANET host names.  They must start with a letter, end with a letter or digit, and have as interior characters only letters, digits, and hyphen.  There are also some restrictions on the length.  Labels must be 63 characters or less.
https://datatracker.ietf.org/doc/html/rfc1034

But the validation library [govalidator](https://github.com/asaskevich/govalidator) defines as follows:
https://github.com/asaskevich/govalidator/blob/f21760c49a8d602d863493de796926d2a5c1138d/patterns.go#L33

It allows hostname to start with a digit.

The "127.0.1" test should result in an error, but this requires modifying the govalidator library.
Since it is the govalidator's responsibility to validate the DNSName correctly, atlantis does not perform this test.